contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -369,8 +369,6 @@ def model_post_init(self, ctx:dict[str,Any]):
         # if not isinstance(director,DirectorOutputDto):
         #     raise ValueError("DirectorOutputDto was not passed in context of Detection model_post_init")
         director: Optional[DirectorOutputDto] = ctx.get("output_dto",None)
-        for story in self.tags.analytic_story:
```

Comment:
Why has this been removed?  This is how we add relevant detections to an analytic story?